test/common.ts

```diff
@@ -497,15 +500,61 @@ describe('logging-common', () => {
       };
 
       loggingCommon.cloudLog[STACKDRIVER_LEVEL] = (
-        entry_: Entry,
+        entry_: Entry[],
         callback: () => void
       ) => {
-        assert.strictEqual(entry_, entry);
+        assert.deepEqual(entry_[0], entry);
```

Comment:
Should the test entry be the first, or should the diagnostic entry come first?  Or does it matter?

Reply:
Since I changed the logic in [LogCommon.log()](https://github.com/googleapis/nodejs-logging-winston/blob/b5c68a8515e2f503b91dba69ee97c3fa77959ac0/src/common.ts#L313) to write array of entries instead a single object, this logic just adopted to perform validations on array of entries